content/en/docs/Customizing/configure_grub.md

```diff
@@ -108,3 +135,37 @@ GRUB_ENTRY_NAME=myOS
 ```
 
 will automatically set the GRUB menu entries for active, passive and recovery to the specified value.
+
+The grub menu boot entry can also be set with `grub2-editenv`:
+
+```bash
+> grub2-editenv /oem/grubenv set default_menu_entry=fooOS
+```
+
+{{% alert title="Additional menu entries" %}}
+
+Since {{<package package="system/grub2-config" >}} >= 0.0.14 it is possible to add multiple custom menu entries to the grub menu by creating a `/grubmenu` config file in one of the available partitions (`COS_STATE`, `COS_PERSISTENT`, `COS_OEM` ) detected by GRUB2. First match wins. The `grubmenu` file will be sourced at the end of the boot process, and can contain several `menuentry` blocks.
```

Comment:
First match wins,but is the order described here the order in which grub loads this stuff? Does it depend on the partition number? Maybe we should clarify

Reply:
Updated!